vm-setup/roles/v1aX_integration_test/vars/main.yml

```diff
@@ -52,13 +52,15 @@ IRONIC_IMAGE_DIR: "{{ lookup('env', 'IRONIC_IMAGE_DIR') | default('/opt/metal3-d
 IRONIC_ENDPOINT_BRIDGE: "{{ lookup('env', 'CLUSTER_PROVISIONING_INTERFACE') | default('provisioning')}}"
 
 # Distibution specific environment variables
+SOURCE_IMAGE_NAME: "{{ lookup('env', 'IMAGE_NAME') | default('bionic-server-cloudimg-amd64.img', true)}}"
 IMAGE_NAME: "{{ lookup('env', 'IMAGE_RAW_NAME') | default('bionic-server-cloudimg-amd64-raw.img', true)}}"
```

Comment:
Is this var still needed? Maybe we could still use the same var `IMAGE_NAME` and replace lookup from `IMAGE_RAW_NAME` to `IMAGE_NAME`?